From eb64faacf3440c004bb101c4e4f767abd26f785a Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Wed, 8 Jul 2026 20:16:08 -0400
Subject: [PATCH] version: drop beta label, bump to Reaper_v1.2

EXTENDNO Reaper_v1.1-beta5 -> Reaper_v1.2. All removal phases 1.1-1.5
metal-validated; this is the first stable cut off the de-bloat line.
---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index 6df1c3bfa67..a4cffd241b7 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v1.1-beta5
+EXTENDNO=Reaper_v1.2
 
 RCNO=0
-- 
2.25.1

